examples/vmseries_scaleset/main.tf

```diff
@@ -14,119 +44,240 @@ resource "random_password" "this" {
   override_special = "_%@"
 }
 
-# Create the network required for the topology.
+# Create the transit network which holds the VM-Series
+# (both inbound and outbound ones).
 module "vnet" {
   source = "../../modules/vnet"
 
   virtual_network_name    = var.virtual_network_name
-  resource_group_name     = azurerm_resource_group.this.name
+  create_virtual_network  = var.create_virtual_network
+  resource_group_name     = local.inbound_resource_group.name
   location                = var.location
   address_space           = var.address_space
```

Comment:
What do you think about using the attribute references for the `location` and `address_space` of a pre-existing Virtual Network, if we decide to use it?

Reply:
Technically it makes sense. But in terms of education, I think it would be just too much for this example.